package/MDAnalysis/analysis/hydrogenbonds/wbridge_analysis.py

```diff
@@ -1210,18 +1209,21 @@ def _donor2acceptor(self, donors, h_donors, acceptor):
                                            box=self.box,
                                            return_distances=True)
         if self.distance_type != 'heavy':
+            tmp_distances = distances
             tmp_donors = [h_donors[donors_idx[idx]] for idx in pairs[:, 0]]
             tmp_hydrogens = [donors_idx[idx] for idx in pairs[:, 0]]
             tmp_acceptors = [acceptor[idx] for idx in pairs[:, 1]]
         else:
             tmp_donors = []
             tmp_hydrogens = []
             tmp_acceptors = []
-            for idx in range(len(pairs[:, 0])):
+            tmp_distances = []
+            for idx, distance in enumerate(distances):
```

Comment:
I think this makes sense, so we don't get lost with this again, could you just add maybe an inline comment about what this is doing here / the importance of this case?